docker/8.0/Dockerfile

```diff
@@ -1,50 +1,101 @@
 FROM composer:2.0.8 as composer
 FROM php:8.0-fpm
 
+# Compilation parameters
 RUN CPU_CORES="$(getconf _NPROCESSORS_ONLN)";
 ENV MAKEFLAGS="-j${CPU_CORES}"
 
 ADD ./extra.ini /usr/local/etc/php/conf.d/
 
-RUN apt update -y && apt install -y \
-    nano \
-    wget \
-    zip \
-    git \
-    apt-utils \
-    sudo \
-    libpq-dev \
-    libmemcached-dev \
-    libmagickwand-dev \
-    libyaml-dev \
-    libicu-dev \
-    libgmp-dev \
-    libpng-dev \
-    libzip-dev && \
-    pecl install psr && \
+# User/Group globals
+ENV MY_USER="phalcon" \
+	MY_GROUP="phalcon" \
+	MY_UID="1000" \
+	MY_GID="1000" \
+	PHP_VERSION="7.4"
+
+# User and Group
+RUN set -eux && \
+	groupadd -g ${MY_GID} -r ${MY_GROUP} && \
+	useradd -u ${MY_UID} -m -s /bin/bash -g ${MY_GROUP} ${MY_USER}
+
+# Update
+RUN apt update -y && \
+    apt install -y \
+        apt-utils \
+        gettext \
+        git \
+        libgmp-dev \
+        libicu-dev \
+        libmagickwand-dev \
+        libmemcached-dev \
+        libpng-dev \
+        libpq-dev \
+        libyaml-dev \
+        libzip-dev \
+        locales \
+        nano \
+        sudo \
+        wget \
+        zip
+
+# PECL Packages
+RUN pecl install psr && \
     pecl install -o -f redis && \
     pecl install igbinary && \
     pecl install msgpack && \
     pecl install apcu && \
     pecl install yaml && \
+    pecl install imagick && \
```

Comment:
There is no PHP8 compatible version for it
https://pecl.php.net/package/imagick

We compile it manually below.